### DISCUSSION TITLE
Help with maintanence

### DISCUSSION THREAD

**Discussion by tmillr** [projekt0n/github-nvim-theme#342] [General]
Hi @ful1e5. I wasn't sure where to post this or how to get in touch with you, so I'll just leave this here. I just wanted to say that I'd be willing to help out with maintaining this project, or even just the issues (e.g. closing invalid issues, or those which are still open but have since been fixed), if that's something that you are interested in. Up to you.

**Comment by ful1e5** [comment]
@tmillr Yes, you can!  Thanks for offering to help. I'm swamped with freelance work right now and can't keep up with everything.  I'll give you access to the repo so you can help out while I'm busy.

**Reply by tmillr** [reply]
Awesome thanks bro! I'll see if I can help out with some things. Feel free to let me know if I do anything you aren't happy with.

**Comment by luisdavim** [comment]
Is this theme still maintained?

**Reply by tmillr** [reply]
Well as for me, I'm still here, I just haven't worked on it in a few months or so. I've been thinking about how we should transition to the new colors, but it's going to be a fairly large change I think. GitHub has renamed or refactored alot of the primitives that we are vendoring and depend on. The 2 largest hurdles imo are that we need to parse them from css, and that it's no longer predictable which colors have alpha (which needs to be accounted for) as it differs by theme.

Are you experiencing any other issues?

**Reply by luisdavim** [reply]
No, no issues. Just wondering if the theme was going to become stale as GitHub gradually changes. 

Thanks for the answer and thank for all the work you've put into this project.

**Reply by luisdavim** [reply]
BTW, regarding the conversion from CSS, wouldn't it be easier to convert the vscode theme?
There's even this tool https://github.com/viniciusmuller/djanho

**Comment by luisdavim** [comment]
Any chances you could have a look at the existing PRs?

**Comment by luisdavim** [comment]
@tmillr @ful1e5 is this project abandoned?
